<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Logexp</class>
 <widget class="QWidget" name="Logexp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>453</width>
    <height>332</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <family>Arial Black</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>Logaritmos:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="log1radioButton">
       <property name="text">
        <string>Log 1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="log2radioButton">
       <property name="text">
        <string>Log 2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="log3radioButton">
       <property name="text">
        <string>Log 3</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <family>Arial Black</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Exponenciales:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QRadioButton" name="exp1radioButton">
       <property name="text">
        <string>Exp 1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="exp2radioButton">
       <property name="text">
        <string>Exp 2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="exp3radioButton">
       <property name="text">
        <string>Exp 3</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="klabel">
       <property name="text">
        <string>k:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="kdoubleSpinBox"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="ButtonCancelar">
       <property name="text">
        <string>Cancelar</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="ButtonAceptar">
       <property name="text">
        <string>Aceptar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <zorder>log1radioButton</zorder>
  <zorder>kdoubleSpinBox</zorder>
  <zorder>klabel</zorder>
  <zorder>ButtonAceptar</zorder>
  <zorder>ButtonCancelar</zorder>
  <zorder>exp2radioButton</zorder>
  <zorder>exp3radioButton</zorder>
  <zorder>log2radioButton</zorder>
  <zorder>log3radioButton</zorder>
  <zorder>exp1radioButton</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>kdoubleSpinBox</zorder>
  <zorder></zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
